### DISCUSSION TITLE
New setup!

### DISCUSSION THREAD

**Discussion by monpiere** [awawa-dev/HyperHDR#774] [General]
I'm about to start a whole new setup on a new 65 inch set in our family room. I have seen the N100 mini PC mentioned on here before to use as the HyperHDR host. My question is, is that the smallest processor I should use, being the alder lake? If not, which specs are important. There are a lot of choices out there! I have an extra adafruit Scorpio I plan on using to drive the LEDs. Thank you all for your help. All the hard work that is put into HyperHDR is much appreciated and doesn't go without notice! 

Thanks again!
Jason

**Comment by awawa-dev** [comment]
Thanks :) In addition to the N100, there is also the N95 from this series, but it is much weaker and the price difference does not justify it. N100 is more powerful than Rpi5. The configuration and quality range of the Chinese N100 is large, usually there are nuances (soldered or replaceable SSD/RAM) and sometimes you may have problems with Windows activation, noisy cooling or the inability to change some parameters in a limited BIOS. Almost everything is OK in my N100, although there was a problem with the license after reinstallation and the coils squeal but only when it is turned off. The only way is to read the comments of people who bought this selected model earlier. Some Linux distro may have also problems with alder lake plaform because it is rather new, but the situation should improve . Windows 11 on N100 works flawlessly with mu USB grabber and DirectX screen capture.

**Reply by monpiere** [reply]
Awesome! Thank you so much for the quick response. Would you mind telling me which model you have and where you got it perhaps? Also, as far as the sk6812 LED strip goes, is 60 led/m dense enough, or would it be beneficial for me to take it up a notch or 2?

**Reply by awawa-dev** [reply]
I have a Firebat AK2. I bought it for less than $100 (including VAT) in the Aliexpress Choice deal. Such deals are quite frequent. Another thing I forgot: be careful with Windows on such cheap Chinese devices, because they may have Trojans installed, so a complete reinstallation of Windows is recommended (but then there may be a problem with license activation). I use 96LEDs/meter and their max. brightness matches very well with the LG C9, which has a fairly high peak brightness. If the TV is not too bright, it is better to choose 60LEDs.
The more dense the LEDs, the higher the maximum brightness (we can control this in HyperHDR) and minimum brightness (and we cannot change this) In the case of Neopixel, more LEDs => lower refresh rate, although this is rather important for large sets.

**Reply by monpiere** [reply]
Perfect! Again, thank you for all the help!
